clients/fides-js/src/fides-tcf.ts

```diff
@@ -236,6 +243,16 @@ const _Fides: FidesGlobal = {
     }
     return this.init(this.config);
   },
+  shouldShowExperience() {
+    if (!isPrivacyExperience(this.experience) || !this.cookie) {
+      throw new Error("Should have an experience and a cookie");
+    }
```

Comment:
Won't this cause `FidesInitialized` to throw errors and never initialize when no experience applies? That's a common use case

Reply:
e.g. https://cookiehouse-plus-rc.fides-staging.ethyca.com/?geolocation=RU

Reply:
It worked okay in my testing because `isPrivacyExperience({}) == true` and `{}` is what the server responds with when there's no specific experience and also what the FE code falls back to if there was an error fetching it. Initially though `this.experience` starts out as `undefined` so if this is called too early, it will result in a (programmer) error, hence this check.

Reply:
Ah OK! Good - I didn't test your branch specifically, I'm just on the hunt for potential breaking changes. Ship it